account_move_template/models/account_move_template.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2015-2017 See manifest
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class AccountMoveTemplate(models.Model):
+    _name = 'account.move.template'
+    _inherit = ['account.document.template',
+                # 'mail.activity.mixin',  TODO:  uncomment for saas-15
+                'mail.thread']
+
+    @api.model
+    def _company_get(self):
+        return self.env['res.company']._company_default_get(
+            object='account.move.template'
+        )
+
+    company_id = fields.Many2one(
+        'res.company',
+        required=True,
+        change_default=True,
+        default=_company_get,
+    )
+    template_line_ids = fields.One2many(
+        'account.move.template.line',
+        inverse_name='template_id',
+    )
+
+    @api.multi
+    def action_run_template(self):
+        self.ensure_one()
+        action = self.env.ref(
+            'account_move_template.action_wizard_select_template').read()[0]
+        action.update({'context': {'default_template_id': self.id}})
+        return action
+
+
+class AccountMoveTemplateLine(models.Model):
+    _name = 'account.move.template.line'
+    _inherit = 'account.document.template.line'
+
+    journal_id = fields.Many2one('account.journal', required=True)
```

Comment:
@rven Can we move the journal_id in account_move_template, and to make this one related, to keep compatibility, no need to select for each line, in case of mistake, you will get with an unbalanced move.

Reply:
Or to have a validation of moves, per journal, as debit=credit

Reply:
@feketemihai This is the way it was designed originally, and we should ask the authors @dcorio. Probably they had a use case that justified the possibility to create multiple journal entries using a single template. It is certainly strange to me, and looks more simple as 1 template == 1 journal entry.

But I would say that if the resulting JE is unbalanced the system is generating an error message at the time of posting:


![image](https://user-images.githubusercontent.com/7683926/48054399-76970280-e1ad-11e8-9a9d-59e439be7b1d.png)